### DISCUSSION TITLE
Installer issue

### DISCUSSION THREAD

**Discussion by skafte** [SierraSoftworks/tailscale-udm#83] [Q&A] [answered]
While installing I get this error: "Installing auto-update timer to ensure that Tailscale is kept installed and up to date.
Failed to enable unit: Unit file tailscale-install.timer does not exist."
Should I force a reinstall, or?

**Comment by notheotherben** [comment, marked as answer]
Hi there, this should be resolved in [v2.5.4](https://github.com/SierraSoftworks/tailscale-udm/releases/tag/v2.5.4), thanks for the report.
